README.md

```diff
@@ -1,381 +1,242 @@
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://neptune.ai/ target=“_blank”>Website</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://docs.neptune.ai/ target=“_blank”>Docs</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://github.com/neptune-ai/examples target=“_blank”>Examples</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://neptune.ai/resources target=“_blank”>Resource center</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://neptune.ai/blog target=“_blank”>Blog</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href=https://neptune.ai/events target=“_blank”>Podcast</a>
+&nbsp;
+  <hr />
 </div>
 
-## Flexible metadata store for MLOps, built for research and production teams that run a lot of experiments.
+## What is neptune.ai?
+
+neptune.ai makes it easy to log, store, organize, compare, register, and share <b>all your ML model metadata in a single place</b>.
+
+* Automate and standardize as your modeling team grows.
+* Collaborate on models and results with your team and across the org.
+* Use hosted, deploy on-premises, or in a private cloud. Integrate with any MLOps stack.
+&nbsp;
 
-Neptune is a lightweight solution designed for:
+&nbsp;
+<div align="center">
+    <a href="https://youtu.be/9jN7RuPNEyc" target="_blank">
+      <img border="0" alt="neptune.ai explainer video" src="https://github.com/neptune-ai/neptune-client/blob/assets/readme/github-video.png" width="600">
+    </a>
+</div>
+&nbsp;
 
-* **Experiment tracking:** log, display, organize, and compare ML experiments in a single place.
-* **Model registry:** version, store, manage, and query trained models and model-building metadata.
-* **Monitoring ML runs live:** record and monitor model training, evaluation, or production runs live.
+&nbsp;
+<a href="https://app.neptune.ai/showcase/example-project-tensorflow-keras/experiments?split=tbl&dash=charts&viewId=eccd5adf-42b3-497e-9cc2-9fa2655429b3" target=“_blank”><b>Play with a live neptune.ai app →</b></a>
 &nbsp;
 
-### Getting started
+&nbsp;
+## Getting started
 
-**Step 1:** Sign up for a **[free account](https://neptune.ai/register)**
+**Step 1:** Create a **[free account](https://neptune.ai/register)**
 
-**Step 2:** Install the Neptune client library
+**Step 2:** Install Neptune client library
 
 ```bash
 pip install neptune-client
 ```
 
-**Step 3:** Connect Neptune to your code
+**Step 3:** Add experiment tracking snippet to your code
 
 ```python
 import neptune.new as neptune
 
-run = neptune.init_run(
-    project="common/quickstarts",
-    api_token=neptune.ANONYMOUS_API_TOKEN,
-)
-
-run["parameters"] = {
-    "batch_size": 64,
-    "dropout": 0.2,
-    "optim": {"learning_rate": 0.001, "optimizer": "Adam"},
-}
-
-for epoch in range(100):
-    run["train/accuracy"].log(epoch * 0.6)
-    run["train/loss"].log(epoch * 0.4)
+run = neptune.init_run("Me/MyProject")
+run["parameters"] = {"lr":0.1,
```

Comment:
`black` should be applied or just reformat this particular snippet. There is a lack of whitespaces before operations.

Reply:
Not sure if I get that? How I can reformat it or apply `black`?

Reply:
Ohh. Just replace it with `{"lr": 0.1, "dropout": 0.4}` (spaces before every number). https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ACtAG5dAD2IimZxl1N_WmXhZA7zE3fzn9obYcWFsOiHVbW-Mluhf2KwGtf8W4pYAywlOxfyq6KX__um_7gmFtjG8I9ZDxnUWYNFs61M8-6xW_ZWBTF2RsLwv3yzLIvl3ox0O12rimPLh1ErKyJk2LIsINwAAAAAqlub-mwJnt8AAYoBrgEAANLOaO2xxGf7AgAAAAAEWVo=

Reply:
thanks, hope it's ok now :)